cmd/subnetcmd/create.go

```diff
@@ -90,7 +92,9 @@ configuration, pass the -f flag.`,
 	cmd.Flags().StringVar(&customVMBranch, "custom-vm-branch", "", "custom vm branch or commit")
 	cmd.Flags().StringVar(&customVMBuildScript, "custom-vm-build-script", "", "custom vm build-script")
 	cmd.Flags().BoolVar(&useRepo, "from-github-repo", false, "generate custom VM binary from github repository")
-	cmd.Flags().BoolVar(&teleporterReady, "teleporter", true, "generate a teleporter-ready vm")
+	cmd.Flags().BoolVar(&useWarp, "warp", true, "generate a vm with warp support (needed for teleporter)")
```

Comment:
nit: wording should be consistent, right now we have `warp` `teleporter` `run-relayer` and the last one should be `relayer`. Another option do `with-warp` `with-releporter` `with-relayer`. Wondering what @sukantoraymond opinion on this. Pls feel free to ignore it

Reply:
using "relayer"